fdb-record-layer-core/src/main/java/com/apple/foundationdb/record/provider/foundationdb/IndexingScrubMissing.java

```diff
@@ -227,7 +234,8 @@ public IndexingScrubMissing(@Nonnull final IndexingCommon common,
                     if (LOGGER.isWarnEnabled() && logWarningCounter > 0) {
                         logWarningCounter --;
                         LOGGER.warn(KeyValueLogMessage.build("Scrubber: missing index entry",
-                                LogMessageKeys.KEY, rec.getPrimaryKey().toString())
+                                        LogMessageKeys.KEY, rec.getPrimaryKey().toString(),
+                                        LogMessageKeys.INDEX_KEY, missingKey.toString())
```

Comment:
This will only log one of the missing keys, but there could be multiple keys that are missing, so we wouldn't have the full scope of the "damage".  Perhaps you want to do something like:

```
AsyncUtil.getAll(indexEntryNoPKs.stream()
                .map(entry -> { ... })
                .filter(Objects::nonNull)
                .collect(Collectors.toList())
                .thenApply(...)
```

this will give you a list of all the missing keys.

Reply:
Fixed.